charts/kubefed/README.md

```diff
@@ -77,7 +77,7 @@ $ helm search kubefed
 Install the chart and specify the version to install with the
 `--version` argument. Replace `<x.x.x>` with your desired version.
 ```bash
-$ helm install fedv2-charts/kubefed --version=<x.x.x> --namespace kube-federation-system
+$ helm install kubefed-charts/kubefed --version=<x.x.x> --namespace kube-federation-system
```

Comment:
This should be `kubefed-charts/federation-v2` for now.

Reply:
Got it.